domain/src/main/kotlin/no/nav/su/se/bakover/domain/behandling/NySøknadsbehandling.kt

```diff
@@ -1,18 +0,0 @@
-package no.nav.su.se.bakover.domain.behandling
-
-import no.nav.su.se.bakover.common.Tidspunkt
-import no.nav.su.se.bakover.domain.behandling.Behandling.BehandlingsStatus
-import no.nav.su.se.bakover.domain.behandling.Behandling.BehandlingsStatus.OPPRETTET
-import no.nav.su.se.bakover.domain.oppgave.OppgaveId
-import java.util.UUID
-
-data class NySøknadsbehandling(
```

Comment:
Må vurdere om vi skal gjøre noen form for håndtering av forskjellig modell inn/ut. Ref hvordan denne gjorde at vi slapp å "hacke" inn Saksnummer ved opprettelse.

Reply:
Jeg synes command-query separation er helt fair i det behandlingstilfellet der. Vi kan gjerne gjeninnføre en lignende type.

Reply:
Denne utsettes til senere (tm)